src/main/java/org/wso2/carbon/connector/FileAppend.java

```diff
@@ -1,5 +1,5 @@
 /*
-* Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+* Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
This file was added in 2016. Therefore the copyright version should be 2016. We do not change the year when we modify it but the year is added when the file is created. Please check all the files.

Reply:
Fixed in https://github.com/wso2-extensions/esb-connector-file/pull/28